qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -183,7 +183,45 @@ def _get_version(self):
             mes = self.ask(bytes([3, 4]))
             ver = mes[2]
             return ver
-
+    
+    def linsweep(self,start,end,points):
+        """
+        A replacement of numpy.linspace for use with the ivvi rack dacs.
+        ivvi.linsweep gives:
+         - even spacing in voltages
+         - voltages match the set voltages
+         - protection from oversampling
+         
+        Args:
+            start (number): starting voltage
+            end (number): ending voltage
+            points (int): number of points between start and end, inclusive
+            
+        Returns:
+            list of valid voltages for the ivvi rack
+            
+        Examples:
+            #normal dac aligned voltages
+            ivvi.linsweep(-200,200,201) -> [ -199.404, -197.391 ... ] 
+            
+            #protection from oversampling
+            ivvi.linsweep(start=108.8,end=109,points=11) -> 
+                [108.82734416723888, 108.88838025482566,108.94941634241245, 109.01045242999923]
+        """
+        points = max(round(abs(points)),2)
+        use_reversed = end < start
+        if use_reversed:
+            start,end = end,start
+        dac_quata = 4.0/65535.0
+        byte_start =  int(round(start/1000.0/dac_quata))
+        byte_end = int(round(end/1000.0/dac_quata))
+        delta_bytes =  abs(byte_end - byte_start)+1
+        spacing =  max(round(delta_bytes / (points-1)),1)
+        l =  [ el*dac_quata*1000 for el in range(byte_start,byte_end+spacing,spacing)]
```

Comment:
please, use spaces for readability
```suggestion
        l =  [el*dac_quata*1000 for el in range(byte_start, byte_end + spacing, spacing)]
```